great_ape_safe/ape_api/badger.py

```diff
@@ -254,3 +255,15 @@ def wire_up_controller(self, controller_addr, vault_addr, strat_addr):
         controller.approveStrategy(want, strat_addr)
         controller.setStrategy(want, strat_addr)
         assert controller.strategies(want) == strat_addr
+
+
+    def set_key_on_registry(self, registry_addr, key, target_addr):
+        registry = interface.IBadgerRegistry(registry_addr, owner=self.safe.account)
```

Comment:
why not add this to the `__init__` func of the class, so that it is always available? also makes passing registry address obsolete.

or do you expect there will be multiple registries...?

Reply:
Noup, good call. Registry V1 is deterministic and the same address on all chains. It may not be the case for registry V2 but we can always adjust it when the time comes.